docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
+
+services:
+  apache-waf:
+    build:
+      context: .
+      dockerfile: v2-apache/Dockerfile
+    environment:
+      BACKEND: http://backend  # https://httpbin.org
+      PORT: "8080"
+      SSL_PORT: "4443"
+    depends_on:
+    - backend
+    user: "10001"
+    ports:
+    - "8080:8080"
+    - "4443:4443"
+
+  nginx-waf:
+    build:
+      context: .
+      dockerfile: v3-nginx/Dockerfile
+    environment:
+      BACKEND: http://backend  # https://httpbin.org
+      PORT: "8080"
+      SSL_PORT: "4443"
+      # DNS_SERVER: "8.8.8.8"
+    depends_on:
+    - backend
+    ports:
```

Comment:
Can we also add the `user: "10001"` here to run it as non-root?

Reply:
Good point! -- That made me discover that for Nginx we don't yet have the OpenShift preparation in place! (`chgrp`, `chmod`)